src/raftstore/store/fsm/peer.rs

```diff
@@ -155,9 +155,9 @@ impl PeerFsm {
     ) -> Result<(LooseBoundedSender<PeerMsg>, Box<PeerFsm>)> {
         // We will remove tombstone key when apply snapshot
         info!(
-            "[region {}] replicate peer with id {}",
-            region_id,
-            peer.get_id()
+            "replicate peer";
+            "region_id" => region_id,
+            "id" => peer.get_id(),
```

Comment:
I prefer s/id/peer_id.

Reply:
Please ignore it since it in peer.rs.

Reply:
Maybe still better to use peer_id to that it can be friendly to be searched in ElasticSearch later.